websocket-chat/public/script.js

```diff
@@ -0,0 +1,75 @@
+const socket = io("http://localhost:3000");
```

Comment:
In the final version of this project learners should also write all of this code. They can start by setting up a basic server then work on this file to get the different functions working.